pkg/api/annotation_key_constants.go

```diff
@@ -19,6 +19,10 @@ limitations under the License.
 package api
 
 const (
+	// ImagePolicyFailedOpenKey is added to pods created by failing open when the image policy
+	// webhook backend fails.
+	ImagePolicyFailedOpenKey string = "alpha.image-policy.k8s.io/failed-open"
```

Comment:
What does the "alpha" in the name refer to? The annotation isn't driving behavior, it's informational. Also, most existing alpha annotations I can find are using `<area>.alpha.kubernetes.io/<name>` (see other annotations in this file)